commands.go

```diff
@@ -514,7 +514,7 @@ func banCMD(line string, u *User) {
 		u.room.broadcast(Devbot, "Which user do you want to ban?")
 		return
 	}
-	victim, ok := findUserByName(u.room, split[0])
+	victim, ok := findUserByIdOrName(u.room, split[0])
```

Comment:
what is the use of this?

Reply:
I feel like it is better to let the admin have the choice. If someone manages in the future to have a malformed name, we would be happy to kick using IDs.

Reply:
That means that this feature is only useful when the name cleaning function has a bug. I think this is a very small use case and that the bug's root cause should be fixed instead when it arises.

Reply:
I totally agree with you on that one, we should always strive to fix the root of an issue. Yet, if such an issue does arise, having an admin do some cleaning on the chatroom before you have time to fix the bug could come in handy.

If you really don't like that feature, I can take it out. The rest of the PR does make sense even without it.

Reply:
I think we can take it out. Super small use case ~~+ I think I fixed all the name bugs already~~

Reply:
Done!